WARG-kicad.pretty/LMZ21701SILT.kicad_mod

```diff
@@ -0,0 +1,22 @@
+(module LMZ21701SILT (layer F.Cu) (tedit 5A7CC1E1)
+  (fp_text reference U15 (at 0 3.81) (layer F.SilkS)
```

Comment:
Can't find a schematic for this footprint, also I don't think the reference field should be anything other than REF**, not U15. This component is also not in the csv, or at least there's no reference of "LMZ21701SILT".